change/@internal-react-composites-b050bf40-8788-4709-bb7a-569a0a94537f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Wouldn't this be prerelease?

Reply:
Is it `non` because all the changes are internal?

Reply:
Kind of prerelease, kinf of none.. the reason I left it as none is the PiP hasn't been added to anything yet - so a changelog that mentions populating it with props won't make sense yet